open-reac/src/test/resources/expected_inputs/param_buses_with_reactive_slack.txt

```diff
@@ -0,0 +1,3 @@
+#amplId powsyblId
```

Comment:
powsyblId :-) everything here is powsybl

Reply:
I agree with this. We have to change the write function of AbstractElementInput class then :)
"#num id" would be consistent with other input/output files. Do you agree ?

Reply:
If it is in core, it is too late...